.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[target.'cfg(target_os = "windows")']
+rustflags = ["-Clink-args=/guard:cf /DYNAMICBASE /CETCOMPAT"]
+
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-Ctarget-feature=+crt-static"]

```

Comment:
Ah, this is a neat way to configure these flags! Do we need similar for other targets as well?

Reply:
Good question, and relates to Ian's question above. I had only seen examples of doing this for Windows and hadn't considered we could just do it across all platforms.